Firestore/Example/Tests/Remote/FSTRemoteEventTests.mm

```diff
@@ -101,52 +145,56 @@ - (void)setUp {
  * part of a previous listen. To modify this set during test execution, invoke
  * `[_targetMetadataProvider setSyncedKeys:forQueryData:]`.
  * @param watchChanges The watch changes to apply before returning the aggregator. Supported
- * changes are FSTDocumentWatchChange and FSTWatchTargetChange.
+ * changes are `DocumentWatchChange` and `WatchTargetChange`.
  */
 - (FSTWatchChangeAggregator *)
-    aggregatorWithTargetMap:(NSDictionary<FSTBoxedTargetID *, FSTQueryData *> *)targetMap
-       outstandingResponses:
-           (nullable NSDictionary<FSTBoxedTargetID *, NSNumber *> *)outstandingResponses
```

Comment:
Just wonder, do we actually pass Nil before this PR? If so, now we pass empty map after the migration, assume?